config.json

```diff
@@ -467,6 +467,15 @@
       ]
     },
     {
+      "slug": "circular-buffer",
+      "difficulty": 8,
+      "topics": [
+        "Queues",
+        "Classes",
+        "Exceptions"
```

Comment:
None of the other exercises have topics at the moment. This has come up recently in #384 and #523 thoughts @exercism/java?